### PR TITLE
Add README for Chinese

### PR DESCRIPTION
**Pull request by Bonjourz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,7 @@
...
+[简体中文](README_CN.md) | English
 
 Welcome to Asterinas Confidential Computing (Asterinas-CC). Asterinas Confidential Computing is a collection of open-source projects featuring full-stack capabilities in confidential computing. Our goal is to provide users with powerful and flexible solutions for confidential and privacy-preserving computing, ensuring the security and privacy of data during processing.
 
-
-# 介绍
-
-星绽机密计算以 HyperEnclave、Occlum 和 TrustFlow 三大核心组件为基础，构建了一个完整的机密计算生态体系。
-
-**HyperEnclave** 作为开放式跨平台可信执行环境（TEE）技术，提供了硬件级别的受保护和隔离的计算环境，并将其信任根与 CPU 厂商解耦。
```
Comment (volcano0dr) [inline README.md:14]: Add  URL link
`[**HyperEnclave**](https://github.com/asterinas/hyperenclave)`

File: README.md [hunk 2]
```diff
@@ -1,24 +1,7 @@
...
-
-**Occlum** 则是一个轻量级的 TEE 操作系统，支持多种主流编程语言和文件系统，使得机密计算应用能够轻松复用现有开发技能和资源。
```
Comment (volcano0dr) [inline README.md:16]: Add URL link
`[**Occlum**](https://github.com/asterinas/occlum)`

File: README.md [hunk 3]
```diff
@@ -1,24 +1,7 @@
...
-
-**TrustFlow** 则专注于构建可信的数据流通和计算框架，确保数据在流转过程中的安全性和隐私性。
```
Comment (volcano0dr) [inline README.md:18]: Add URL link
[**TrustFlow**](https://github.com/asterinas/trustflow)

File: README.md (hunk 2 above)
Comment (Bonjourz) [reply README.md:16]: Done.

File: README.md (hunk 3 above)
Comment (Bonjourz) [reply README.md:18]: Done.

File: README.md (hunk 1 above)
Comment (Bonjourz) [reply README.md:14]: Done.

**Comment by volcano0dr** [comment]
LGTM

**Pull request merged**